lib/elixir_console_web/templates/console/history/index.html.leex

```diff
@@ -0,0 +1,10 @@
+<div class="flex-1"></div>
+  <div class="p-2" id="commandOutput" phx-update="append">
+    <%= for output <- @output do %>
+      <div id="command<%= output.id %>" class="text-gray-300 font-medium"><%= print_prompt() %><%= format_command(output.command) %></div>
+      <div id="output<%= output.id %>" class="text-teal-300">
+        <%= if output.result do output.result end %>
```

Comment:
Question out of ignorance, what would happen if you just do `<%= output.result %>` and it's not defined? wouldn't it ignore it? (or render nothing to be more precise)

Reply:
That's a good question that I can not answer off the top of my head.
Let me try it and confirm.

Reply:
You're totally right... nothing is rendering if the value is nil, so the conditional is useless